README.md

```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
 });
 ```
 
+#### Open files preview dialog
+Opens a popup card with the preview of an asset
+
+**type**
+`'openFilesDialog'`
+
+**params**
+
+| Parameter|Type | Description | Required | Default Value |
+| --- | --- | --- | --- | --- |
+| boardId| Integer | The ID of the board | Yes | |
+| itemId| Integer | The ID of the item, which contains an asset | Yes | |
+| columnId| String | The ID of the column, which contains an asset | Yes | |
+| assetId| Integer | The ID of the asset to open | Yes | |
+
+**Example**
+```javascript
+monday.execute('openFilesDialog', {
```

Comment:
we get assetIds from the params - is it generic or only docs use case?

![image](https://user-images.githubusercontent.com/17691874/115736798-3e430900-a394-11eb-9d8f-5734e8290e1c.png)

Reply:
I don't know, but it doesn't work for me on prod + I think it's not very clear UI and I don't see much use cases for it